pkgs/by-name/xa/xan/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "xan";
+  version = "0.42.0";
+
+  src = fetchFromGitHub {
+    owner = "medialab";
+    repo = "xan";
+    tag = version;
+    hash = "sha256-8uNfiPJjZWqtbrBU/LRYM7PI5/4JBqZGe1dtCWArU8M=";
+  };
+
+  cargoHash = "sha256-4OVDiXa8iutoPr816yjMFYa+bC5vZ+iHHrij7uwYUZ0=";
```

It might be better to use `useFetchCargoVendor` here as it's less likely to have its hashes invalidated (see https://github.com/NixOS/nixpkgs/pull/376038): 

```suggestion
  cargoHash = "sha256-TRT//USKHCc7MAgNWAiZoovQmkud3isT+kwsNkJH8Bk=";
  useFetchCargoVendor = true;
```